W-18090420-no-adv-alerts-duke (#367)
mulesoft/docs-monitoring: modules/ROOT/pages/_partials/include-alerts-config.adoc

> = Create an Alert for a Custom Dashboard

Author: ```suggestion
= Create an Alert for a Chart in a Custom Dashboard
```

[[alerts]]

. Navigate to the configuration screen for a graph in a custom dashboard. For guidance, see xref:dashboard-custom-config.adoc[Configuring Custom Dashboards].
. Provide a general configuration for the graph that (at minimum) defines the metric, environment, and resource for which you want to set the alert. 
+
For guidance, see xref:dashboard-custom-config-graph.adoc#general_settings[General Settings].

> . Navigate to the *Alert* configuration tab for the graph.

Author: Below is the main difference. Everything else is basically changing the language in the doc from "advanced alert" to clarify that we're talking about alerts for charts in custom dashboards.

[[notifications]]
== Notifications

Configure an email notification when an alert is triggered.

[%header,cols="1,4"]
|===
2+| Notification Settings for Alerts

> | `Send To` | Email address for each recipient of the alert notification. If you add or change the email address of an alert recipient, they do not receive an email until the alert is triggered.

Reviewer: ```suggestion
| `Send To` | Email address for each recipient of the alert notification. If you add or change the email address of an alert recipient, they don't receive an email until the alert is triggered.
```
